toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/panels/SimplePanel.java

```diff
@@ -0,0 +1,35 @@
+package io.toast.tk.agent.ui.panels;
+
+import io.toast.tk.agent.ui.utils.PanelHelper;
+import org.apache.logging.log4j.LogManager;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.logging.log4j.LogManager'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)